packages/squiggle-lang/__tests__/ReducerProject/ReducerProject_tutorial_0_test.res

```diff
@@ -0,0 +1,109 @@
+@@warning("-44")
```

Comment:
Why does this file start as tutorial_0_test instead of tutorial_1_test ? Is that a programming thing? My hunch is that starting with 1 would be simpler. 

(Sometimes 0 is meant as "before the start", if there was something special about that, but I don't think that's how it's meant here)